### PR TITLE
fix: authentication animation

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

File: packages/utils-react/src/key-animation/__styles__/key-animation.ts [hunk 1]
```diff
@@ -69,7 +72,7 @@ export const keyhole = css``
 export const keyholeMask = css`
   width: 2.5rem;
   height: 3.5rem;
-  background: var(--color-dark-blue);
+  background: #262f68;
```
Comment (bashleigh) [inline packages/utils-react/src/key-animation/__styles__/key-animation.ts:75]: some reason the var didn't work here making the mask invisible 🤷‍♀️  not sure why but this resolves the problem

**Review by joshbalfour — APPROVED** [review]

File: packages/utils-react/src/key-animation/__styles__/key-animation.ts (hunk 1 above)
Comment (willmcvay) [reply packages/utils-react/src/key-animation/__styles__/key-animation.ts:75]: You got the variable wrong!
Screenshot 2021-11-02 at 12 40 41

Comment (willmcvay) [reply packages/utils-react/src/key-animation/__styles__/key-animation.ts:75]: Referring to your docs 😂

Comment (bashleigh) [reply packages/utils-react/src/key-animation/__styles__/key-animation.ts:75]: 🙄 typical 😂

Comment (bashleigh) [reply packages/utils-react/src/key-animation/__styles__/key-animation.ts:75]: Screenshot 2021-11-02 at 13 27 07
Still doesn't work!

**Pull request merged**